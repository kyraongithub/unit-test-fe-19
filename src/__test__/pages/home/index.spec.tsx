import Home from '@/components/pages/homepage';
import { render } from '@testing-library/react';

describe('Home Page', () => {
  it('render homepage component', () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
