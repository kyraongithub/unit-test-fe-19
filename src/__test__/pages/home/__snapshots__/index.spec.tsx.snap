// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page render homepage component 1`] = `
<div>
  <div>
    <h1>
      Home
    </h1>
    <button />
  </div>
</div>
`;
